infrastructure/nomad-configuration/client-instance-user-data.tpl.sh

```diff
@@ -13,6 +13,37 @@
 # provisioners which will put files onto the instance after it starts up,
 # but those run after this script runs.
 
+# Mount the EFS.
+export DEBIAN_FRONTEND=noninteractive
+apt-get update -y
+apt-get upgrade -y
+apt-get install --yes nfs-common
+mkdir -p /var/efs/
+echo "${file_system_id}.efs.${region}.amazonaws.com:/ /var/efs/ nfs4 nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 0 0" >> /etc/fstab
+mount -a -t nfs4
+chown ubuntu:ubuntu /var/efs/
+
+# Set up the database
+# XXX - Failure here is catastrophic - even normally "safe" operations like apply permissions to a user that already has them
+# XXX - We need to have way to do this "manually" for staging and automatically for prod, all without
+# XXX - the catastrophic failure.
+apt-get install --yes postgresql-client-common postgresql-client
```

Comment:
Not sure this is all even necessary any more.

We could use Packer avoid needing do a lot of this stuff every time but I don't want to open that portal to hell right now.